### ISSUE TITLE
Accessibility: ListPeoplePicker input role should be 'textbox' not 'combobox'

### ISSUE THREAD

**Issue by daweffe** [OfficeDev/office-ui-fabric-react#2374]
### Bug Report

- __Package version(s)__: 4.21.3
- __Browser and OS versions__: All

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Normal

Products/sites affected: Planner web

#### Describe the issue:

Screen readers announce the ListPeoplePicker input text box as a combo box, which is confusing.

#### Actual behavior:

role is combobox

#### Expected behavior:

role should be textbox

**Comment by micahgodbolt** [comment]
@daweffe give me a shout if you need a hand with that PR.

**Comment by daweffe** [comment]
@micahgodbolt It might be easier for you to submit a PR for this.

**Comment by micahgodbolt** [comment]
Other than the lack of a down arrow, the people picker does behave a lot like a combobox 

http://oaa-accessibility.org/examplep/combobox2/

If anything, I could see us adding aria-autocomplete=“inline”.

Why do you feel that textbox is more appropriate?

**Comment by daweffe** [comment]
The people picker as a whole acts like a combobox.  But the input text box inside it is just a textbox.  It shouldn't be a combobox in a combobox.

**Comment by daweffe** [comment]
You're right.  I will consult the accessibility team and close this bug for now.
